analytics/analytics.go

```diff
@@ -32,14 +32,14 @@ func buildStatus(buildFailed bool) string {
 	return map[bool]string{false: "successful", true: "failed"}[buildFailed]
 }
 
-func stepStatus(i int) string {
-	if status, ok := map[int]string{
+func stepStatus(status models.StepRunStatus) string {
```

Comment:
Breaking change in the models package, the enum values got a typealias to an int.